.github/CODEOWNERS

```diff
@@ -1,14 +1,14 @@
 # Global Reviewer
-.github @rancher/mapps
-docs @rancher/mapps
-scripts @rancher/mapps
-CNAME @rancher/mapps
-Makefile @rancher/mapps
-README.md @rancher/mapps
-_config.yml @rancher/mapps
-configuration.yaml @rancher/mapps
-regsync.yaml @rancher/mapps
-index.html @rancher/mapps
+/.github @rancher/mapps
```

Comment:
Works the same as the roof files. With the slash first  `/.github`. This means that even if there were subdirectories containing a `.github` file or folder, it wouldn't trigger a review from us because we only own the one in the root directory "/.github".